### DISCUSSION TITLE
Uploading files to the agency swarm through the chatbot interface.

### DISCUSSION THREAD

**Discussion by trees777** [VRSEN/agency-swarm#65] [Q&A]
Hello all, I am so happy I have found this project and have been trying to build demos to share with leads of my work/team! I have a couple of use cases at the moment that I am trying to build out but am reaching a dead end when coming to figure out how to upload a file to the swarm... two ideas that I can't hack:

1. Trying to upload an excel, get the swarm to interact with and edit it, and then redownload onto my computer.
2. Trying to upload a pdf to be reviewed by and gain insights from the swarm.

If anyone can help or point me in the right direction, I would be so grateful!!! Just trying to take these ideas a step further and really build some impactful tools here.

Thank you and take care.

**Comment by MizpahMJP** [comment]
Hi 
Did you get a solution to this? Would really be useful if you could share. Thanks

**Comment by nicko-ai** [comment]
This may be helpful: https://github.com/VRSEN/agency-swarm/issues/161
